.github/workflows/build-ubuntu.yml

```diff
@@ -0,0 +1,55 @@
+name: Build on Ubuntu
+
+on:
+  push:
+    branches:
+      - master
+      - v*
```

Comment:
I don't think it needs to be specified twice. Just use `tags` and it will suffice.

Reply:
Yes and no. We don't currently tag stable-backports. But we also don't really merge into stable branches either. I agree it may not be necessary. Artifacts don't persist longer than 90 days, so we will need actual release builds for github releases anyway.

Reply:
Actually, I'll leave it because we may want to get a preview build of a stable branch before actually cutting out a tag from it.